src/client/BaseApi.ts

```diff
@@ -189,6 +231,31 @@ export class BaseApi implements IAstarApi {
         }
     }
 
+    public async getStakerInfo(address: string): Promise<bigint> {
+        await this.ensureConnection();
```

Comment:
Main reason for having this endpoint is to get info for EVM stakers. You are missing address conversion.
If address provided is H160 than you need to check if account is unified (use `unifiedAccounts` pallet). If the is unified use mapped SS58 address. If the account is not unified use SS58 pair (see `toSS58Address` in `astar.js`)
See https://github.com/AstarNetwork/astar-apps/blob/e8e3852b7a7cb4191a60a27df009b96b43fca552/src/hooks/useAccount.ts#L44